docs/modules/ROOT/pages/overview.adoc

```diff
@@ -0,0 +1,61 @@
+= Overview
+:page-aliases: docs:ROOT:overview.adoc
+
+Trustification is composed of several independent base services as well as services that provide combined functionality from those base services.
+
+* Bombastic - Storage, archival, search and exporting of Software Bill Of Materials (SBOM) documents.
+* Vexination - Storage, archival, search and exporting of Vulnerability EXchange (VEX) documents.
+* Exhort - Generating reports and recommendations for your dependencies.
+* V11y - Database for recording and retrieving information about a vulnerability.
+* GUAC - Graph for Understanding Artifact Composition. Provides a graph of relationships between artifacts and vulnerabilities.
```

Comment:
Wanna mention collector[ist]? It's kinda GUAC-related.

Reply:
We should, but I also presumed that's on my shoulders to write up.

Reply:
I don't know what the collectorist does so feel free to add that!